spec/services/create_account_verified_event_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+describe CreateVerifiedAccountEvent do
+  let(:password) { 'password' }
```

Comment:
The User Factory already creates a password. For this test, all that matters is that the Event is tied to a user. This should be sufficient:
```ruby
let(:user) do
  create(:user) do |user|
    user.events.create(event_type: :account_verified)
  end
end

let(:eventless_user) { create(:user) }
```